druid/src/widget/scroll.rs

```diff
@@ -97,7 +97,9 @@ impl<T, W: Widget<T>> Scroll<T, W> {
     ///
     /// Returns `true` if the scroll offset has changed.
     pub fn scroll(&mut self, delta: Vec2, layout_size: Size) -> bool {
-        self.scroll_component.scroll(delta, layout_size)
+        let scrolled = self.scroll_component.scroll(delta, layout_size);
```

Comment:
Feels like an interaction that other potential users of scroll component are going to get wrong. Not sure if theres a better way though..

Reply:
Yeah, I'm not wild about the fact that it needs to be handled in three separate places. Maybe I'll find some time next weekend to take a stab at a `Viewport`...